src/metabase/cmd/env_var_dox.clj

```diff
@@ -82,6 +86,20 @@
        ;; Drop brackets used to create source code links
        (#(str/replace % #"\[\[|\]\]" ""))))
 
+(defn- deprecated-message
+  "For shouting (and listing a message, like the version it was deprecated, if available)."
+  [env-var]
+  (let [deprecated (:deprecated env-var)]
+    (if (string? deprecated)
+      (str "> DEPRECATED: " deprecated)
+      "> DEPRECATED")))
+
+(defn- format-deprecated
+  "Tags a deprecated environment variable."
+  [env-var]
+  (when (:deprecated env-var)
+    (deprecated-message env-var)))
+
```

Comment:
Any reason to keep around both of these functions? they do almost the same thing and could be combined

```suggestion
(defn- format-deprecated
  "Tags a deprecated environment variable."
  [env-var]
  (when-let [deprecated (:deprecated env-var)]
    (if (string? deprecated)
      (str "> DEPRECATED: " deprecated)
      "> DEPRECATED")))
```